Players/libvlc-rtsp-player/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
Is it important to store gradlew and wrapper?

Reply:
Could be useful to someone

Reply:
Isn't that for those who can't generate them?